psydac/cad/tests/test_geometry.py

```diff
@@ -195,9 +195,9 @@ def test_export_nurbs_to_hdf5(ncells, degree):
     eta2 = refine_array_1d( new_pipe.breaks(1), 10 )
 
     pcoords1 = np.array( [[new_pipe(e1,e2) for e2 in eta2] for e1 in eta1] )
-    pcoords2 = np.array( [[mapping([e1,e2]) for e2 in eta2] for e1 in eta1] )
+    pcoords2 = np.array( [[mapping(e1,e2) for e2 in eta2] for e1 in eta1] )
```

Comment:
Any hope that this will work with Numpy broadcasting? Something like the following:
```suggestion
    ee1, ee2 = np.meshgrid(eta1, eta2, indexing='ij', sparse=True)
    pcoords1 = new_pipe(ee1, ee2)
    pcoords2 =  mapping(ee1, ee2)
```

Reply:
It doesn't because `TensorFemSpace.eval_field` doesn't accept arrays because `psydac.core.bsplines.find_span` and `basis_funs` do not either.

Reply:
Right! I forgot that these are spline mappings...